innovation.game.php

```diff
@@ -11652,6 +11706,47 @@ function stInteractionStep() {
             );
             break;            
 
+        // id 190, Artifacts age 8: Meiji-Mura Stamp Vending Machine
+        case "190N1A":
+            // Return a card from your hand.
+            $options = array(
+                'player_id' => $player_id,
+                'can_pass' => false,
+                'n' => 1,
```

Comment:
To be consistent with other card I always put the "n" between "player_id" and "can_pass"

Reply:
sure